.gitignore

```diff
@@ -71,3 +71,5 @@ test/selenium/screenshots/*
 
 # logging
 logs/*.log*
+
+edx-ora2/
```

Comment:
I think that this is another place logs can appear, though I'm confused about why this wasn't already included here.